freanalysis_clouds/freanalysis_clouds/__init__.py

```diff
@@ -5,56 +5,32 @@
 from analysis_scripts import AnalysisScript
 from figure_tools import Figure, LonLatMap
 import intake
-import intake_esm
 
 
 @dataclass
 class Metadata:
-    activity_id: str = "dev"
-    institution_id: str = ""
-    source_id: str = ""
-    experiment_id: str = "c96L65_am5f7b11r0_amip"
-    frequency: str = "P1M"
-    modeling_realm: str = "atmos"
-    table_id: str = ""
-    member_id: str = "na"
-    grid_label: str = ""
-    temporal_subset: str = ""
-    chunk_freq: str = ""
-    platform: str = ""
-    cell_methods: str = ""
-    chunk_freq: str = "P1Y"
-
-    def catalog_search_args(self, name):
-        return {
-            "experiment_id": self.experiment_id,
-            "frequency": self.frequency,
-            "member_id": self.member_id,
-            "modeling_realm": self.modeling_realm,
-            "variable_id": name,
-        }
+    """Helper class that stores the metadata needed by the plugin."""
+    frequency: str = "monthly"
```

Comment:
Looks good. In the future we'll probably update this to "mon" as we constrain the frequency with

https://github.com/NOAA-GFDL/CMIP6_CVs/blob/main/CMIP6_frequency.json